db/flags.sql

```diff
@@ -1,5 +1,20 @@
 CREATE SCHEMA IF NOT EXISTS flags;
 
+-- TODO: should this also have a column for qc_time or some such?
+CREATE TABLE IF NOT EXISTS flags.confident_provenance (
+    timeseries INT4 NOT NULL,
+    obstime TIMESTAMPTZ NOT NULL,
+    pipeline TEXT NOT NULL,
+    -- TODO: should this be an enum?
```

Comment:
Does this comment refer to the line below or the one below? Suggest to move the comment to after the line to remove ambiguity.